Soomla/Assets/Soomla/Editor/build-tools/Soomla_CoreRunner.py

```diff
@@ -30,11 +33,22 @@
 pbx_file_path = sys.argv[1] + '/Unity-iPhone.xcodeproj/project.pbxproj'
 pbx_object = XcodeProject.Load(pbx_file_path)
 
+soomla_binaries_path = path.join(script_dir, '..', '..', '..', 'Plugins', 'iOS', 'SOOMLA')
+target_soomla_binaries_path = path.join(build_path, 'Libraries', 'SOOMLA')
+if not os.path.isdir(target_soomla_binaries_path):
+  os.mkdir(target_soomla_binaries_path)
+soomla_libraries_group = pbx_object.get_or_create_group('SOOMLA', parent=pbx_object.get_or_create_group('Libraries'))
+for library in os.listdir(soomla_binaries_path):
+  name_components = library.split('.')
+  if (name_components[len(name_components) - 1] == 'a'):
+    copyfile(path.join(soomla_binaries_path, library), path.join(target_soomla_binaries_path, library))
+    pbx_object.add_file_if_doesnt_exist(path.join(target_soomla_binaries_path, library), tree='SOURCE_ROOT', parent=soomla_libraries_group)
+
 for framework in frameworks:
   pbx_object.add_file_if_doesnt_exist(framework, tree='SDKROOT')
 
 for framework in weak_frameworks:
-  pbx_object.add_file_if_doesnt_exist(framework, tree='SDKROOT', weak=True)
+  pbx_object.add_file_if_doesnt_exist(framework, tree='SDKROOT', weak=True)\
```

Comment:
@borisspektor, sure, I didn't find it during testing because it's meanless in logic. I'll remove it.